src/style.css

```diff
@@ -1,3 +1,84 @@
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  background-color: lightgrey;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  min-width: 1024px;
+  padding: 0 50px;
+}
+
+.nav_ul {
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  align-content: center;
+  list-style-type: none;
+  margin: 0;
+}
+
+.logo_box {
+  display: block;
+  height: 40px;
+}
+
+.logo_img {
+  width: 40px;
+  height: 40px;
+}
+
+.nav_list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  padding: 0;
+  margin-right: 20px;
+  line-height: 15px;
+}
+
+.nav_list:last-child {
+  margin-right: 0;
+}
+
+.nav_link {
+  position: relative;
+  color: #000;
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-decoration: none;
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
+  line-height: 15px;
```

Comment:
do nothing here

Reply:
what do you mean? 1) text-decoration doing my text without underline$ 2) paddings + line-height make all height to 60px